### PR TITLE
[new release] ocaml-protoc (4 packages) (3.0.1)

### PR DESCRIPTION
**Pull request by c-cube**
Pure OCaml compiler for .proto files

- Project page: <a href="https://github.com/mransan/ocaml-protoc">https://github.com/mransan/ocaml-protoc</a>

##### CHANGES:

- fix bug where code for services would be generated even if `--services`
    is not passed on the CLI

### CODE REVIEW

File: packages/ocaml-protoc/ocaml-protoc.3.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+depends: [
+  "dune" {>= "2.0"}
+  "odoc" {with-doc}
+  "pbrt" {= version}
+  "pbrt_yojson" {= version & with-test}
+  "pbrt_services" {= version & with-test}
+  "ocaml" {>= "4.08"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/ocaml-protoc/ocaml-protoc.3.0.1/opam:18]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/pbrt/pbrt.3.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+homepage: "https://github.com/mransan/ocaml-protoc"
+bug-reports: "https://github.com/mransan/ocaml-protoc/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "stdlib-shims"
+  "odoc" {with-doc}
+  "ocaml" {>= "4.08"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/pbrt/pbrt.3.0.1/opam:16]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/pbrt_services/pbrt_services.3.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+homepage: "https://github.com/mransan/ocaml-protoc"
+bug-reports: "https://github.com/mransan/ocaml-protoc/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08"}
+  "pbrt" {= version}
+  "pbrt_yojson" {= version}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/pbrt_services/pbrt_services.3.0.1/opam:16]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/pbrt_yojson/pbrt_yojson.3.0.1/opam [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+bug-reports: "https://github.com/mransan/ocaml-protoc/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08"}
+  "odoc" {with-doc}
+  "yojson" {>= "1.6"}
+  "base64" {>= "3.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/pbrt_yojson/pbrt_yojson.3.0.1/opam:18]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by mseri** [comment]
Thanks

**Comment by c-cube** [comment]
Thank you :)

**Pull request merged**